### DISCUSSION TITLE
Make the GitBook icon easier to see

### DISCUSSION THREAD

**Discussion by gewaleelek** [Arcticons-Team/Arcticons#2251] [Ideas]
It's almost invisible in GitHub dark mode 😅

![image](https://github.com/user-attachments/assets/828f36b3-e89d-49da-ba60-e11ee1b1716c)

**Comment by gewaleelek** [comment]
I tried to make the new GitBook button in the style of other buttons in the readme:

![gitbook](https://github.com/user-attachments/assets/e7a00fcd-c87d-40f2-ab1f-81114a6a62ee)

The GitBook icon comes from [the official site](https://gitbook.gitbook.io/brand-assets), but I did change the stroke color and width. I'm not sure if that is prohibited by whatever GitBook license we are under.

Any thoughts/suggestions?

**Comment by Donnnno** [comment]
Sorry, finally updated it!
